shared/js/sk_key_utils.js

```diff
@@ -0,0 +1,367 @@
+(function (global) {
+  'use strict';
+
+  var softKeyModule = (function () {
+
+    // Keys which are mapped to accesskey
+    const SK_KEYS_MAPPING = {
+      LSK: 'a',
+      CSK: 's',
+      RSK: 'd'
+    };
+
+    // Storing the reference to softkeys
+    var sk_keys = {
```

Comment:
This object stores references to the DOM elements that the soft keys reference.  A name like sk_elements might be clearer than sk_keys.

And since this is all wrapped up inside a private function, there isn't any need for sk_ prefixes here. The code might be simpler to read without them.